### PR TITLE
docs(gatsby-theme-bodiless, vital-image): Add info about static Vital Images

### LINKED ISSUE #1803: [DOC] Vital Image is Static
## User Story
AS AN end user of the Bodiless documentation,
I WANT to know about the static nature of Vital Images,
SO THAT, I can be confident in the performance of image-rendering on my site.

## Acceptance Criteria
1. The Documentation is updated to include information about Vital Images being static.

### References

#1452

### PR DESCRIPTION
**Pull request by misterjones**
:warning: **This branch — `docs/static-vital-image` — is based off a branch that has since merged to `main` — `docs/vital-hero`.**

**See:** PR #1792 

----

## Changes
- Update "Vital Image Component" page with info about Vital Images being static.
- Clean up `gatsby-theme-bodiless` documentation.
- Update `gatsby-theme-bodiless` package to generated documentation.
- Update missed instances of "cx" and "CanvasX" (and variants thereof) to "Vital".

## Related Issues

Fixes #1803

### CODE REVIEW

**Comment by hvanyo** [comment]
@misterjones  Can you replace the vital card example with this..

The following EXTENDS the vitalCard.WithVerticalOrientation and adds the following border & padding to the wrapper.
`
const WithVerticalCardBorder = asCardToken(vitalCard.WithVerticalOrientation, {
  Theme: {
    Wrapper: 'border border-black'
  },
  Spacing: {
    Wrapper: 'px-4',
  },
});
// Then compose the Vital Default Card with new custom vertical style and node key.
const VerticalCardWithBorder = as(
  vitalCard.Default,
  WithVerticalCardBorder,
  withNodeKey('card-vertical-example'),
)(CardClean);
`
The following OVERRIDES the vitalCard.WithVerticalOrientation, it uses all the same domains from WithVerticalOrientation, except spacing, and these are replaced with larger paddings.  
`
const WithVerticalOrientationMoreSpacing = asCardToken({
  ...vitalCard.WithVerticalOrientation,
  Spacing: {
    ContentWrapper: 'p-8',
    ImageWrapper: 'p-8',
  },
});
// Then compose the Vital Default Card with new custom vertical style and node key.
const VerticalCardWithSpacing = as(
  vitalCard.Default,
  WithVerticalOrientationMoreSpacing,
  withNodeKey('hero-card-secondary-button'),
)(CardClean);
`

File: packages/gatsby-theme-bodiless/doc/README.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+#### Omit Gatsby Plugin Image for Static Image
+
+When using Gatsby Image with `withoutHydration` from `@bodiless/hydration`, and providing a static
+version, `gatsby-plugin-image` is no longer required on the front-end. The code can be removed from
+the final bundle by setting the environment variable `BODILESS_GATSBY_PLUGIN_IMAGE_OMIT` to `1`.
+
+### Plugins
+
+#### Robots.txt
```
Comment (hvanyo) [inline packages/gatsby-theme-bodiless/doc/README.md:145]: @misterjones  Can you link 17 Robots.txt to this section: https://johnsonandjohnson.github.io/Bodiless-JS/#/Development/Guides/BuildingSites/

Never knew we had this documented.

Can you Shemas to boidless schemas doc as well.

File: packages/vital-editors/doc/RichText.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ defined in the Schema Domain.
 
 ### Available Rich Text Editors
 
-There are some Editor Tokens available on cxRichText.
+There are some Editor Tokens available on `vitalRichText`.
```
Comment (hvanyo) [inline packages/vital-editors/doc/RichText.md:43]:
```suggestion
There are some Editor Tokens available in `vitalRichText`.
```

File: packages/vital-image/doc/README.md [hunk 3]
```diff
@@ -45,6 +45,15 @@ The Vital Image Component provides a set of tokens to compose an Image Component
 | `WithLink`                 | A token which wraps the image in a link.                     |
 | `WithFullWidthImage`       | A token which makes the image full-width.                    |
 
+### Static Images
```
Comment (hvanyo) [inline packages/vital-image/doc/README.md:48]: Can you link #4 to vital-image doc in https://johnsonandjohnson.github.io/Bodiless-JS/#/Development/Guides/BuildingSites/

File: packages/gatsby-theme-bodiless/doc/README.md (hunk 1 above)
Comment (misterjones) [reply packages/gatsby-theme-bodiless/doc/README.md:145]: @hvanyo I hadn't realized it was documented either — this package wasn't generating docs, so it was "hidden."

**Review by marcopagliarulo — APPROVED** [review]

**Pull request merged**